apisix/plugins/ldap-auth.lua

```diff
@@ -30,13 +30,12 @@ local schema = {
     title = "work with route or service object",
     properties = {
         base_dn = { type = "string" },
-        ldap_host = { type = "string" },
-        ldap_port = { type = "number" },
+        ldap_uri = { type = "string" },
```

Comment:
Deleting these two options seems incompatible, can't we keep them?

Reply:
What do you mean? Those are just new config items which make it incompatible, so I revert to use the old one.

Reply:
I thought `ldap_host` and `ldap_port` became `ldap_uri` now, and I was wondering if they were compatible. For example, there is now an old copy of the data in etcd and they contain `ldap_host` and `ldap_port`. when updating the APISIX version, the user needs to reset the `ldap_uri`?

Reply:
`ldap_uri` is the original configuration item. Nothing change here.

Reply:
got it.